site/docs/reference/Connectors/capture-connectors/youtube-analytics.md

```diff
@@ -0,0 +1,80 @@
+
+This connector is based on an open-source connector from a third party, with modifications for performance in the Flow system.
+You can find their documentation [here](https://docs.airbyte.com/integrations/sources/youtube-analytics/),
+but keep in mind that the two versions may be significantly different.
+
+## Supported data resources
+
+The following data resources are supported through the YouTube Analytics APIs:
+
+* [channel_annotations_a1](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#video-annotations)
+* [channel_basic_a2](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#video-user-activity)
+* [channel_cards_a1](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#video-cards)
+* [channel_combined_a2](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#video-combined)
+* [channel_demographics_a1](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#video-viewer-demographics)
+* [channel_device_os_a2](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#video-device-type-and-operating-system)
+* [channel_end_screens_a1](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#video-end-screens)
+* [channel_playback_location_a2](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#video-playback-locations)
+* [channel_province_a2](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#video-province)
+* [channel_sharing_service_a1](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#video-content-sharing)
+* [channel_subtitles_a2](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#video-subtitles)
+* [channel_traffic_source_a2](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#video-traffic-sources)
+* [playlist_basic_a1](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#playlist-user-activity)
+* [playlist_combined_a1](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#playlist-combined)
+* [playlist_device_os_a1](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#playlist-device-type-and-operating-system)
+* [playlist_playback_location_a1](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#playlist-playback-locations)
+* [playlist_province_a1](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#playlist-province)
+* [playlist_traffic_source_a1](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#playlist-traffic-sources)
+
+By default, each resource is mapped to a Flow collection through a separate binding.
+
+## Prerequisites
+
+* YouTube does not start to generate a report until you create a [reporting job](https://developers.google.com/youtube/reporting/v1/reports#step-3:-create-a-reporting-job) for that report.
```

Comment:
Seems like you copied this from Airbyte's docs verbatim. This isn't advisable for a few reasons:
1. Plagiarism - we should be rephrasing everything we get from Airbyte.
2. It mentions "Airbyte" and we are "Flow."
3. Often, Airbyte's docs are structured in a different way than ours are. They're also pretty poorly written. This is a great example of that: not only is this paragraph difficult to read and doesn't actually _say much_ that's relevant to a user, it's not really a prerequisite. It's just telling us what the connector will do. If you look at Airbyte's docs page for this connector, you'll notice that what we list as "prerequisites" are baked into their "setup" sections. The convention we've been following in our docs thus far is to provide a bulleted list of Prerequisites, with an optional Setup section if meeting those prerequisites isn't very straightforward.

Reply:
I just tried the youtube connector and didn't need to prep anything else other than the oauth authentication. I think we can remove this prerequisite.